docs/manual/replacements_and_deletions.md

```diff
@@ -42,11 +42,45 @@ The following conditions are currently eligible for replacement:
 * `MissingPod`: This indicates a process group that doesn't have a Pod assigned.
 * `MissingPVC`: This indicates that a process group that doesn't have a PVC assigned.
 * `MissingService`: This indicates that a process group that doesn't have a Service assigned.
-* `PodPending`: This indicates that a process group where the pod is in a pending state.
+* `PodPending`: This indicates that a process group where the Pod is in a pending state.
+* `NodeTaintReplacing`: This indicates that a process group where the pod is running on a tainted Node
```

Comment:
```suggestion
* `NodeTaintReplacing`: This indicates that a process group where the pod is running on a tainted Node.
```

This is only half true, if the node is not tainted anymore the ProcessGroup will be replaced anyway (at least in the current implementation) if the `NodeTaintReplacing` condition is set. We should add this details here. The implication there is that if a ProcessGroup has the `NodeTaintReplacing` condition the replacement cannot be stopped, even if the node was fixed.